specs/UserDataFolder.md

```diff
@@ -0,0 +1,102 @@
+
+Title
+===
+UserDataFolder API 
+
+# Background
+You may have a need to cleanup or possibly add to the data being
+stored in the WebView2 user data directory.  This directory can be either 
+configured by you during creation of the webview2 control or calculated
+at runtime by the control if it wasn't set during creation.
+
+The current WebView2 code is designed to use a the directory that the hosting 
+application is running in as the default location.  This has resulted in issues 
```

Comment:
"that the hosting application is running" is ambiguous. Could mean the current directory, or could mean the application directory. Should clarify

> The current WebView2 code defaults the user data folder to the directory that contains the hosting application's executable.

I reordered the words to avoid having a really long clause inserted into the middle of a sentence.